### ISSUE TITLE
question about dataset

### ISSUE THREAD

**Issue by YuBeomGon** [openai/supervised-reptile#29]
In the paper, I found this sentence,
 Unlike FOMAML, Reptile doesn’t need a training-test split for each task, which
may make it a more natural choice in certain settings
(https://arxiv.org/pdf/1803.02999.pdf)

it means I dont need to split data?(support set and query set in each episode in train data set) 
Am I right?

**Comment by unixpickle** [comment]
I think you are interpreting the sentence correctly. Basically, Reptile doesn't require you to split up each mini-dataset during meta-learning. It seems that FOMAML works mostly _because_ there's a separate mini-test-set that has a useful gradient after the model has fully overfit to the mini-training-set.
